src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -98,16 +98,16 @@ public abstract partial class ECDsa : System.Security.Cryptography.AsymmetricAlg
     }
     public struct ECParameters
     {
-        public System.Security.Cryptography.ECCurve Curve;
-        public byte[] D;
-        public System.Security.Cryptography.ECPoint Q;
+        public System.Security.Cryptography.ECCurve Curve { get; set; }
```

Comment:
I had a chat with @nguerrera about this, and he's boosted my confidence in saying that having the struct-within-a-struct be a property is kind of crazy expensive (especially given how large ECCurve is).

So, both because 1) we already checked it in as fields, and changing it requires staging in a new contract, and 2) it's expensive, we should just revert this back to the fields.  (Nick looked appalled at my "make the reference types be properties but the value types be fields" compromise).

Sorry for vacillating, this should be the last time :).